packs/ansible/actions/galaxy.install.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: "galaxy.install"
+description: "Download & Install role from ansible galaxy"
+runner_type: "local-shell-script"
+entry_point: "ansible_galaxy.py"
+enabled: true
+parameters:
+  kwarg_op:
+    description: "Lock operator type to '--'"
+    type: string
+    default: "--"
+    immutable: true
+  sudo:
+    description: "Lock sudo, the behavior is controlled by ansible 'become_' options"
+    type: boolean
+    default: true
+    immutable: true
+  timeout:
+    description: "Action timeout in seconds. Action will get killed if it doesn't finish in timeout seconds"
+    type: integer
+    default: 300
+  cwd:
+    description: "Working directory where the command will be executed in"
+    type: string
+  action:
+    description: "Action to use"
+    type: string
+    position: 0
+    default: install
+    immutable: true
+  roles:
```

Comment:
However when running st2 action without any optional parameters: 

```sh
st2 run ansible.galaxy.install

id: 556301bb9c9938107a5c7074
status: failed
result:
{
    "failed": true,
    "stderr": "Executed command "/opt/stackstorm/virtualenvs/ansible/bin/ansible-galaxy install None"
",
    "return_code": 1,
    "succeeded": false,
    "stdout": "- downloading role 'None', owned by
- sorry, None was not found on galaxy.ansible.com.
- you can use --ignore-errors to skip failed roles.
"
}
```

So `st2 run ansible.galaxy.install` is transformed into `ansible-galaxy install None` which I don't think good behavior from st2 engine. Positional parameter still could be optional.